requirements.txt

```diff
@@ -0,0 +1 @@
+-e .
```

Comment:
Typo?

Reply:
Since our fork now contains some changes, the required package is just our local fork, which (if you're in the `immuta/tap-slack` directory) can be installed with `pip install .`.  I was actively developing this tap, so I used the ["editable" install](https://pip.pypa.io/en/stable/reference/pip_install/#editable-installs) (`pip install -e .`), which just means that I do not have to re-install the package in my environment every time I make a change. It automatically updates the environment anytime the `tap_slack` package is edited.

This `-e .` line is equivalent to line 9 in the `pipfile`.